src/libraries/System.Private.Xml/src/System/Xml/Schema/SchemaAttDef.cs

```diff
@@ -100,7 +103,7 @@ string IDtdDefaultAttributeInfo.DefaultValueExpanded
 
         object IDtdDefaultAttributeInfo.DefaultValueTyped
         {
-            get { return ((SchemaAttDef)this).DefaultValueTyped; }
+            get { return ((SchemaAttDef)this).DefaultValueTyped!; }
```

Comment:
Why this can't be null?

And also, why `SchemaDeclBase.DefaultValueTyped` can be null? All references seem to be assigning a not-null reference.

Reply:
seems it can be null and code is fine with that, I must have thought I already annotated the interface (there are couple of similarly named in the same file) and assumed implementation is doing something weird